src/lib/Behat/PageElement/AdminList.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+use Behat\Mink\Exception\ElementNotFoundException;
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+
+class AdminList extends Element
+{
+    protected $fields = ['list' => 'form',
+                        'plusButton' => '.ez-icon-create',
+                        'trashButton' => '.ez-icon-trash',
+                        'editButton' => 'tr:nth-child(%s) a[title=Edit]',
+                        'listHeader' => '.ez-table-header__headline',
+                        'tableHeader' => 'th',
+                        'listElementLink' => '.ez-checkbox-cell+ td a',
+                        'tableCell' => 'tr:nth-child(%s) td:nth-child(%s)', ];
+
+    /** @var string Name by which Element is recognised */
+    public const ELEMENT_NAME = 'Admin List';
+    protected $listHeader;
+
+    public function __construct(UtilityContext $context, string $listHeader)
+    {
+        parent::__construct($context);
+        $this->listHeader = $listHeader;
+    }
+
+    private function verifyProperList(): void
+    {
+        if ($this->listHeader !== $this->context->findElement($this->fields['listHeader'], $this->defaultTimeout)->getText()) {
```

Comment:
Can we extract `$this->context->findElement($this->fields['listHeader'], $this->defaultTimeout)->getText()` to method or variable to improve readability?